core/src/main/java/hudson/model/DownloadService.java

```diff
@@ -46,7 +46,6 @@
 import java.net.URLEncoder;
 import java.util.ArrayList;
 import java.util.List;
-import java.util.concurrent.TimeUnit;
```

Comment:
@daniel-beck : this line's removal needs to be unremoved.

Reply:
Fixed.